### PR TITLE
fix: review runtime

### PR DESCRIPTION
**Pull request by nud3l**

### CODE REVIEW

File: parachain/runtime/kintsugi/src/lib.rs [hunk 1]
```diff
@@ -460,9 +460,10 @@ parameter_types! {
     pub const LaunchPeriod: BlockNumber = 7 * DAYS;
     pub const VotingPeriod: BlockNumber = 7 * DAYS;
     pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
-    // TODO: this needs to change before launch. First off, CENTS is for KSM, not for KINT. Second, needs to be sufficciently
-    // high to prevent dos by making 100 proposals
-    pub MinimumDeposit: Balance = 100 * CENTS;
+    // Require 5 vKINT to make a proposal. Given the crowdloan airdrop, this qualifies about 3500
+    // accounts to make a governance proposal. Only 2300 can do two proposals,
+    // and 700 accounts can do ten or more proposals.
+    pub MinimumDeposit: Balance = 5 * UNITS;
```
Comment (sander2) [inline parachain/runtime/kintsugi/src/lib.rs:466]: The units of the different currencies is very error prone, I think we should add helper functions like [this](https://github.com/AcalaNetwork/Acala/blob/master/runtime/common/src/lib.rs#L153-L167)

File: parachain/runtime/kintsugi/src/lib.rs [hunk 2]
```diff
@@ -121,7 +119,9 @@ pub mod token_distribution {
     use super::*;
 
     // 10 million KINT distributed over 4 years
-    pub const INITIAL_ALLOCATION: Balance = 10_000_000_000_000_000_000;
+    // KINT has 12 decimal places, same as KSM
+    // See: https://wiki.polkadot.network/docs/learn-DOT#kusama
+    pub const INITIAL_ALLOCATION: Balance = 10_000_000_u128 * UNITS;
```
Comment (gregdhill) [inline parachain/runtime/kintsugi/src/lib.rs:124]: I think this is now wrong, we should calculate from `UNITS` or add the helper as @sander2 suggests.

**Review by gregdhill — CHANGES_REQUESTED** [review]

**Review by gregdhill — APPROVED** [review]

File: parachain/runtime/kintsugi/src/lib.rs (hunk 1 above)
Comment (gregdhill) [reply parachain/runtime/kintsugi/src/lib.rs:466]: Let's drop a ticket in the backlog to do this :+1:

Comment (nud3l) [reply parachain/runtime/kintsugi/src/lib.rs:466]: Yeah, I agree. Let's do that in a backlog ticket

**Pull request merged**